data/mods/Aftershock/player/bionics.json

```
@@ -63,6 +63,86 @@
     "react_cost": "12 kJ",
     "time": 1
   },
+  {
+    "id": "bio_forcefield_bash_cut_weak",
+    "type": "bionic",
+    "name": { "str": "Weak Gravity Manipulation Field" },
+    "description": "Imagine a computer inside your body playing pong against the universe.  The paddles are made of gravity fields and how it causes this to happen are trade secrets.  -From the owners manual of HawkingCorps GravyField Mark I.",
+    "occupied_bodyparts": [ [ "torso", 8 ] ],
+    "available_upgrades": [ [ "bio_forcefield_bash_cut_medium" ], [ "bio_forcefield_bash_cut_heavy" ] ],
```

@anothersimulacrum  I don't understand why this one is failing while others in same format are not.

---

The other ones in the same format are not because the game stops loading at this error. The game excepts a simple list of unique bionic ids.
```suggestion
    "available_upgrades": [ "bio_forcefield_bash_cut_medium", "bio_forcefield_bash_cut_heavy" ],
```